src/app/onboarding/components/create-new-archive/create-new-archive.component.html

```diff
@@ -1,7 +1,99 @@
 <div>
-  <h2>Create Your First Archive</h2>
-  <p>Give your archive a name and choose what kind of archive it is.</p>
-  <ng-container>
-    <pr-new-archive-form (success)="onSubmit($event)" (error)="onError()"></pr-new-archive-form>
-  </ng-container>
+  <div *ngIf="screen === 'type'">
+    <h2>Create Your First Archive: Select Type</h2>
+
+    <p>What will your first archive represent? It might be yourself, a family member, an ancestor, your neighborhood association, a small business or a nonprofit organization.</p>
+
+    <p>Not sure what archive option to choose? Start with yourself. You can always create new archives or change the name and type of your archive later.</p>
+
+    <p><strong>What type of archive will this be?</strong> You can always change this later.</p>
+
+    <div class="archive-type-selector">
+      <label>
+        <div [ngClass]="getArchiveTypeClasses('type.archive.person')">
+          <img src="assets/svg/onboarding/archive-person.svg" alt="Person Archive" />
+          <div>
+            <strong class="archive-type-name">Person Archive</strong>
+            <span class="archive-type-description">represents an individual person, such as yourself or someone else</span>
+          </div>
+          <input type="radio" name="archive-type" value="type.archive.person" (change)="setArchiveType($event.target.value)" />
+        </div>
+      </label>
+
+      <label>
+        <div [ngClass]="getArchiveTypeClasses('type.archive.family')">
+          <img src="assets/svg/onboarding/archive-group.svg" alt="Group Archive" />
+          <div>
+            <strong class="archive-type-name">Group Archive</strong>
+            <span class="archive-type-description">any group of people, such as a community group or a family</span>
+          </div>
+          <input type="radio" name="archive-type" value="type.archive.family" (change)="setArchiveType($event.target.value)" />
```

Comment:
Should these `<input type="radio">`s have a CSS class of some sort? I would expect them to be hidden (`display: none;`), but they're showing up for me in Firefox:

![Screen Shot 2022-02-24 at 12 01 31](https://user-images.githubusercontent.com/1494855/155598602-6246868e-0339-4663-9625-d1d6f0ee75e2.png)

They are properly enclosed in `<label>`s, so I can click on the pretty pictures to switch between them, which is great!

The Figma illustration has the containing elements update the background to indicate the selected option. If that's too complicated, maybe we can center them (which is always trivial to do, right?~) for a first draft, and plan to implement the shading later?

Reply:
Okay I have to look into what is actually going on, because I have style rules for hiding the inputs, updating background, centering things, and dividing everything properly but they're just... not showing up on your end.

For the record, here is what I see:
![2022-02-24-131712](https://user-images.githubusercontent.com/9145247/155600791-db8eb5a6-d027-4a62-b51a-9406b7e866a6.png)

Reply:
Oh no! Then all these comments are super noisy, sorry!

Reply:
Resolving this; more in a main-thread comment to come.